contest-service/service.bal

```diff
@@ -96,6 +96,25 @@ type SubmissionData record {|
     |} challenge;
 |};
 
+type registrant record {|
+    string id;
+    record {|
+        string id;
+    |} contest;
+    record {|
+        string id;
+        string fullname;
+    |} user;
+    time:Civil registeredTime;
+|};
+
+type registrantOut record {
```

Comment:
We usually use the entity name for the return type specified in `GET` requests. I would use `Registrant` instead of `registrantOut`. 

For the POST payload, my suggestion is to use `NewRegistrant`